src/mouse-touchpad.vala

```diff
@@ -125,7 +125,7 @@ namespace MouseTouchpad {
 
         private bool has_mouse (Gdk.DeviceManager manager) {
             foreach (var device in manager.list_devices (Gdk.DeviceType.SLAVE)) {
-                if (device.get_source () == Gdk.InputSource.MOUSE && !device.get_name ().has_prefix ("Virtual core")) {
+                if (device.get_source () == Gdk.InputSource.MOUSE || device.get_source () == Gdk.InputSource.TRACKPOINT && !device.get_name ().has_prefix ("Virtual core")) {
```

Comment:
```suggestion
                if ((device.input_source == Gdk.InputSource.MOUSE || device.input_source == Gdk.InputSource.TRACKPOINT) && !device.get_name ().has_prefix ("Virtual core")) {
```
there is a logic issue here:
`&&` has the priority over `||` so you need to put a parenthesis